docs/use_cases/rag_cli.md

```diff
@@ -0,0 +1,109 @@
+# RAG CLI
+
+One common use case is chatting with an LLM about files you have saved locally on your machine.
+
+We have written a CLI tool do help you do just that very easily! You can point the rag CLI tool to a set of files you've saved locally, and it will ingest those files into a local vector database that is then used for a Chat Q&A repl within your terminal.
+
+By default, this tool uses OpenAI for the embeddings & LLM as well as a local Chroma Vector DB instance. **Warning**: this means the local data you ingest with this tool _will_ be sent to OpenAI's API.
```

Comment:
we should note that these dependencies are completely customizable, don't want to scare anyone off right away 👍🏻